core/src/com/mygdx/game/client/SocketIOClient.java

```diff
@@ -78,11 +78,11 @@ private void configureSocketEvents() {
                 .on(SocketEventConstants.SOCKET_ID, getSocketIdEventListener())
                 .on(SocketEventConstants.NEW_PLAYER_CONNECTED, getNewPlayerConnectedEventListener())
                 .on(SocketEventConstants.PLAYER_DISCONNECTED, getPlayerDisconnectedEventListener())
-                .on(SocketEventConstants.GET_PLAYERS, getGetPlayersEventListener())
-                .on(SocketEventConstants.PLAYER_MOVED, getPlayerMovedEventListener());
+                .on(SocketEventConstants.GET_PLAYERS, getGetExistingPlayersEventListener())
+                .on(SocketEventConstants.PLAYER_MOVED, getPlayerUpdatedEventListener());
```

Comment:
I think it might be due to the issue not being descriptive enough but the intended change was actually to rename all constants, methods and internal event bus events referring to these two events to the suggested names.
So you'd need to make changes in 
1. https://github.com/sagdutt/Raze-LibGdx/blob/master/core/src/com/mygdx/game/event/events/PlayerMovedEvent.java
2. https://github.com/sagdutt/Raze-LibGdx/blob/master/core/src/com/mygdx/game/event/events/GetPlayersEvent.java
3. https://github.com/sagdutt/Raze-LibGdx/blob/master/core/src/com/mygdx/game/constant/SocketEventConstants.java#L15-L17
4. https://github.com/sagdutt/Raze-LibGdx/blob/master/core/src/com/mygdx/game/screen/ArenaScreen.java#L182-L202

Reply:
So, you want each occurence of PlayerMoved to be updated to PlayerUpdated and each occurence of GetPlayers to be updated to GetExistingPlayers.

Reply:
Yes exactly

Reply:
Is this not gonna be done? We should reassign it, as it has been sitting idle for a long time.